examples/ecs/ecs_guide.rs

```diff
@@ -239,6 +239,17 @@ fn local_state_system(mut state: Local<State>, query: Query<(&Player, &Score)>)
     state.counter += 1;
 }
 
+#[derive(Debug, Hash, PartialEq, Eq, Clone, StageLabel)]
+enum MyStages {
```

Comment:
As mentioned in another comment, I think we should be consisent with stage enum naming. Imo this should be `MyStage` for parity with `CoreStage`